app/lib/handleZcapRequest.ts

```diff
@@ -0,0 +1,96 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { WAS_KEYS, zcap_expires } from '../../app.config';
```

Comment:
```suggestion
import { WAS_KEYS, ZCAP_EXPIRES } from '../../app.config';
```
We don't use lowercase snake case for variables. We only either use SNAKE_CASE in uppercase for constants, or `camelCase` for variables.

Reply:
got it